pkg/k8s/containers/security.go

```diff
@@ -76,3 +81,11 @@ type WindowsOptions struct {
 	GMSACredentialSpec     string
 	RunAsUserName          string
 }
+
+func (wo *WindowsOptions) toK8S() *v1.WindowsSecurityContextOptions {
```

Comment:
should return nil if nothing is configured

Reply:
done